### PR TITLE
Copy Periodic Background Sync explainer

### PR DESCRIPTION
**Pull request by mugdhalakhani**
from  https://github.com/beverloo/periodic-background-sync to this
repository for wider visibility for feedback.

Once this has been checked in, it'll be deleted from
https://github.com/beverloo.

### CODE REVIEW

**Comment by mugdhalakhani** [comment]
Ping!

File: explainers/periodicsync-explainer.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+# Periodic Background Sync
+
+**Written**: 2019-03-26<br/>
+**Updated**: 2019-04-17
```
Comment (beverloo) [inline explainers/periodicsync-explainer.md]: update this

File: explainers/periodicsync-explainer.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+Periodic Background Sync is a method that enables web applications to periodically synchronize data
+in the background, building on the [Background Sync](https://wicg.github.io/BackgroundSync/spec/)
+specification that defines a method for one-off synchronization.
+
+The [original explainer](https://github.com/WICG/BackgroundSync/blob/master/explainer.md#periodic-synchronization-in-design)
```
Comment (beverloo) [inline explainers/periodicsync-explainer.md]: Update the link.

File: explainers/periodicsync-explainer.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+included a thorough exploration of this capability, but it neither got specified, nor ever shipped
+in a browsing engine. This was caused by a combination of low interest from both developers and
+other browser engines, and high complexity due to an unclear permission model. We believe the
+situation has changed since.
+
+If this proposal stands, we plan to extend the [Background Sync](https://wicg.github.io/BackgroundSync/spec/)
+specification to describe this ability as well, and move the document to the standardization track.
+
+## Use cases
+  Consider a website that's offline enabled. It uses service workers to provide an almost
```
Comment (beverloo) [inline explainers/periodicsync-explainer.md]: Why is this section indented? Only list items have to be indented.

File: explainers/periodicsync-explainer.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+* Guaranteed cadence of the periodic sync tasks. The browser decides this cadence for each origin.
+An origin can thus register multiple periodic tasks, but the frequency decided by the browser for
+the tasks might be the same.
+* Shared namespace for one-shot and periodic sync tasks is not a goal. The same tag can thus be used
+for a one-shot and a periodic sync task. However, two one-shot Background Sync tasks or two periodic
+Background Sync tasks can't share a tag.
+
+# Example code
+
+Please see [WebIDL.md](WebIDL.md) for the proposed WebIDL.
```
Comment (beverloo) [inline explainers/periodicsync-explainer.md]: Update link

File: explainers/periodicsync-explainer.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+registration doesn't have to be the same.
+
+# References and acknowledgements
+
+  * [Background Sync](https://wicg.github.io/BackgroundSync/spec/)
+  * [Push API](https://w3c.github.io/push-api/)
+
+Many thanks to the authors of the original explainer for starting the project, and to @mugdhalakhani
+for picking periodic sync up again. Thanks to @jakearchibald and @rayankans for their ideas, input
+and discussion.
```
Comment (beverloo) [inline explainers/periodicsync-explainer.md]: Please update the acknowledgements, you should be the primary author, but keep mentioning the others.

Comment (mugdhalakhani) [reply explainers/periodicsync-explainer.md]: done

File: explainers/periodicsync-explainer.md (hunk 1 above)
Comment (mugdhalakhani) [reply explainers/periodicsync-explainer.md]: done.

File: explainers/periodicsync-explainer.md (hunk 2 above)
Comment (mugdhalakhani) [reply explainers/periodicsync-explainer.md]: done.

File: explainers/periodicsync-explainer.md (hunk 3 above)
Comment (mugdhalakhani) [reply explainers/periodicsync-explainer.md]: fixed.

File: explainers/periodicsync-explainer.md (hunk 4 above)
Comment (mugdhalakhani) [reply explainers/periodicsync-explainer.md]: done.

**Comment by mugdhalakhani** [comment]
@beverloo, I've made the changes you requested, on the latest version of the explainer from your repo. PTAL.

**Pull request merged**